### ISSUE TITLE
Hand off ownership if abandoned?

### ISSUE THREAD

**Issue by acahir** [WickyNilliams/headroom.js#311]
Seems lots of people still find this utility useful, but with 9 PRs waiting and 2 years since update, could it be handed off? 

If no response from owner, should we create a new "official" fork? Just looking through the PRs there's a few important ones in there.

**Comment by liquidvisual** [comment]
This library is dead, are there any alternatives?

**Comment by WickyNilliams** [comment]
You are right, I have not been doing my job well as maintainer. Life gets in the way sometimes.

There are some PRs outstanding, but nothing major in my opinion. What do you see as important? 

I am not keen on handing over ownership after the recent [event-stream debacle](https://blog.npmjs.org/post/180565383195/details-about-the-event-stream-incident). However, I would be open to having a few maintainers help out. I would always be available to advise/comment on PRs issues etc.

**Comment by WickyNilliams** [comment]
PS. I never understood the attitude that if there are no commits, a project is dead. It still works perfectly. Having 9 PRs on a project with > 10k stars is definitely not a bad state of affairs

**Comment by liquidvisual** [comment]
My biggest issue with Headroom at the moment is the `headroom--bottom` class doesn't get applied on iOS devices, it's broken.

**Comment by WickyNilliams** [comment]
I don't have an iOS device, so I am not able to debug that. And nobody has proposed a PR to fix

**Comment by lkraav** [comment]
@WickyNilliams https://www.browserstack.com/open-source might be a solution.

**Comment by WickyNilliams** [comment]
Update: i'm hoping to get some help with maintenance very soon, so the lib doesn't stagnate further. I'll post back as the situation improves

**Comment by WickyNilliams** [comment]
Well the people who said they'd help never did...

@hacknug, you've been incredibly helpful in responding to issues in the past. Would you like to assist? I'm trying to go through and get all issues/PRs cleaned up now, so everything will be in a good state

**Comment by hacknug** [comment]
@WickyNilliams sure thing! Just let me know how can I help you 👍

**Comment by WickyNilliams** [comment]
@hacknug amazing. I really appreciate that!

For now, i think just helping out in issues/PRs is enough (which I see you've been doing already), until I get everything in order. I am happy to give you permissions to merge/push/close issues etc once I get to that point, if you would like?

I am away from tomorrow, so will pick things back up early next week.

**Comment by WickyNilliams** [comment]
Hey everyone,

I just published `v0.10.0` to npm. This contains a whole host of perf improvements, bug fixes, and features. Along with a modernisation of the code base and toolchain.

Here's the changelog: http://wicky.nillia.ms/headroom.js/#v0.10.0

And for those too lazy to click, here's a screengrab :)

![image](https://user-images.githubusercontent.com/1091390/64700903-af73c800-d49f-11e9-9e70-6cac5bc79fb3.png)

Over the past month I've closed 29 issues, and merged 8 PRs: https://github.com/WickyNilliams/headroom.js/pulse/monthly

I hope you will all now agree I can confidently close this issue!
